ui/lib/kv/addon/templates/secret/metadata/index.hbs

```diff
@@ -1 +1,6 @@
-<Page::Secret::MetadataDetails @model={{this.model}} @breadcrumbs={{this.breadcrumbs}} />
\ No newline at end of file
+<Page::Secret::MetadataDetails
+  @model={{this.model.metadata}}
+  @path={{this.model.path}}
```

Comment:
I called this `@secretPath` - I don't have a preference for either but I think we should be consistent. I can update on my end if you'd like to keep it `@path`

Reply:
The models, adapters and serializers all refer to it as path. Are you okay if we go with that?